scripts/g.extension/g.extension.py

```diff
@@ -2200,6 +2200,20 @@ def resolve_source_code(url=None, name=None, branch=None):
                 url_validated = True
             except:
                 pass
+            # Test if github repo exists (need to use API)
+            try:
+                open_url = urlopen('https://api.github.com/repos/{}/{}'.format(*url.split("/")[-2:]))
+                open_url.close()
+                url_validated = True
+            except:
+                pass
+            # Test if gitlab repo exists (need to use API)
+            try:
+                open_url = urlopen('https://gitlab.com/api/v4/projects/{}%2F{}'.format(*url.split("/")[-2:]))
```

Comment:
I'm using double quotes in python string interpolation in case `open_url = urlopen("https://api.github.com/repos/{}/{}".format(` and simple quotes in case `*url.split('/')`.